etc/indico.conf.sample

```diff
@@ -300,8 +300,8 @@ MaxUploadFileSize = 0
 # OAUTH PARAMETERS
 #------------------------------------------------------------------------------
 
-# Time to live of the OAuth Tokens
-OAuthAccessTokenTTL = 10000
+# Time to live of OAuth grant tokens (in seconds)
+OAuthGrantTokenTTL = 100
```

Comment:
Since it's not used except for a very short time during the oauth process (it's not the actual token lifetime after all), I don't see why an admin would ever need to change it